core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/diagnostics/model/DiagnosticsCollectionRequest.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.diagnostics.model;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.authorization.annotation.ResourceObjectField;
+import com.sequenceiq.authorization.resource.AuthorizationResourceAction;
+import com.sequenceiq.authorization.resource.AuthorizationVariableType;
+import com.sequenceiq.cloudbreak.doc.ModelDescriptions;
+import com.sequenceiq.common.api.diagnostics.BaseDiagnosticsCollectionRequest;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("DiagnosticsCollectionRequest")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DiagnosticsCollectionRequest extends BaseDiagnosticsCollectionRequest {
+
+    @NotNull
+    @ApiModelProperty(ModelDescriptions.StackModelDescription.CRN)
+    @ResourceObjectField(action = AuthorizationResourceAction.DATALAKE_READ, variableType = AuthorizationVariableType.CRN)
```

Comment:
what happens if you use this on distrox api? i think it will fail because it check for datalake

Reply:
Yea I think it would fail. I don't think it's a good idea to check datahub operations with datalake resource type

Reply:
ohh sry its an old comment, not relevant, browser kept it